assets/scss/modules/_preload.scss

```diff
@@ -51,3 +51,45 @@
     animation: flash 2s infinite linear;
   }
 }
+
+/* loading kpi */
+
+.wrapper-widget{
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 8em;
+  height: 8em;
+
+  .loading-box {
+    position: relative;
+    display: flex;
+    top: 50%;
+    align-items: center;
+    justify-content: center;
+    color:$dp-color-lightgrey;
+
+    &:before {
+      @include dp-preloader($dp-color-lightgrey, 5em);
```

Comment:
❤️ 
One thing that comes to my mind looking at the deployed version, is that for this mixin we could move the spinner to the `::after` pseudo-element. Why?
By doing so, we could pass a new parameter to `dp-preloader` mixin with a hexa color value to use as a background that could cover the whole block in where this preloader is going to be used to truly hide the content until it gets hidden or removed from the DOM. We could create a `::before` pseudo-element (if the background parameter exists) in a fixed position to cover the container. 

Let's give it a thought. 👌

Reply:
we could add it as an improvement, it's a good idea, I need to think it well